### PR TITLE
reference/system-databases: add 3 cluster tables (2)

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)

Add the `cluster_load`, `cluster_systeminfo` and `cluster_log` system tables.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

**If you select two or more versions from above**, to trigger the bot to cherry-pick this PR to your desired release version branch(es), you **must** add corresponding labels such as **needs-cherry-pick-4.0**, **needs-cherry-pick-3.1**, **needs-cherry-pick-3.0**, and **needs-cherry-pick-2.1**.

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/2522
- Other reference link(s):

### CODE REVIEW

File: reference/system-databases/cluster-systeminfo.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+| NAME        | varchar(256) | YES  |      | NULL    |       |
+| VALUE       | varchar(128) | YES  |      | NULL    |       |
++-------------+--------------+------+------+---------+-------+
+6 rows in set (0.00 sec)
+```
+
+Field description:
+
+* `TYPE`: Corresponds to the `TYPE` field in the [`information_schema.cluster_info`](/reference/system-databases/cluster-info.md) table. The optional values are `tidb`, `pd`, and `tikv`.
+* `INSTANCE`: Corresponds to the `STATUS_ADDRESS` field in the [`information_schema.cluster_info`](/reference/system-databases/cluster-info.md) cluster information table.
```
Comment (reafans) [inline reference/system-databases/cluster-systeminfo.md:34]:
```suggestion
* `INSTANCE`: Corresponds to the `INSTANCE` field in the [`information_schema.cluster_info`](/reference/system-databases/cluster-info.md) cluster information table.
```

Comment (reafans) [reply reference/system-databases/cluster-systeminfo.md:34]: This has been changed to `INSTANCE `in new version.

File: reference/system-databases/cluster-load.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+| DEVICE_NAME | varchar(64)  | YES  |      | NULL    |       |
+| NAME        | varchar(256) | YES  |      | NULL    |       |
+| VALUE       | varchar(128) | YES  |      | NULL    |       |
++-------------+--------------+------+------+---------+-------+
+```
+
+Field description:
+
+* `TYPE`: Corresponds to the `TYPE` field in the [`information_schema.cluster_info`](/reference/system-databases/cluster-info.md) table. The optional values are `tidb`, `pd`, and `tikv`.
+* `INSTANCE`: Corresponds to the `STATUS_ADDRESS` field in the [`information_schema.cluster_info`](/reference/system-databases/cluster-info.md) cluster information table.
```
Comment (reafans) [inline reference/system-databases/cluster-load.md:33]: ditto

Comment (lilin90) [reply reference/system-databases/cluster-load.md:33]: @reafans I see that [the Chinese version](https://pingcap.com/docs-cn/dev/reference/system-databases/cluster-load/) still uses `STATUS_ADDRESS` in `cluster-load.md`. Would you please check and update all necessary places?

File: reference/system-databases/cluster-load.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
+---
+title: CLUSTER_LOAD
+summary: Learn the `CLUSTER_LOAD` cluster load table.
+category: reference
+---
+
+# CLUSTER_LOAD
+
+The `CLUSTER_LOAD` cluster load table provides the current load information of the server where each node of the TiDB cluster is located.
```
Comment (lilin90) [inline reference/system-databases/cluster-load.md:9]: @reafans Please help confirm whether it's `instance` or `node` here.
```suggestion
The `CLUSTER_LOAD` cluster load table provides the current load information of the server where each instance of the TiDB cluster is located.
```

File: reference/system-databases/cluster-load.md (hunk 2 above)
Comment (lilin90) [inline reference/system-databases/cluster-load.md:33]:
```suggestion
* `INSTANCE`: Corresponds to the `INSTANCE` field in the [`information_schema.cluster_info`](/reference/system-databases/cluster-info.md) cluster information table.
```

File: reference/system-databases/cluster-log.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
+---
+title: CLUSTER_LOG
+summary: Learn the `CLUSTER_LOG` cluster log table.
+category: reference
+---
+
+# CLUSTER_LOG
+
+You can query cluster logs on the `CLUSTER_LOG` cluster log table. By pushing down query conditions to each node, the impact of the query on cluster performance is less than that of the `grep` command.
```
Comment (lilin90) [inline reference/system-databases/cluster-log.md:9]: @reafans Please help confirm whether it's `instance` or `node` here and other places in this file.
```suggestion
You can query cluster logs on the `CLUSTER_LOG` cluster log table. By pushing down query conditions to each instance, the impact of the query on cluster performance is less than that of the `grep` command.
```

File: reference/system-databases/cluster-log.md [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+| MESSAGE  | var_string(1024) unsigned | YES  |      | NULL    |       |
++----------+---------------------------+------+------+---------+-------+
+5 rows in set (0.00 sec)
+```
+
+Field description:
+
+* `TIME`: The time to print the log.
+* `TYPE`: The node type. The optional values are `tidb`, `pd`, and `tikv`.
+* `INSTANCE`: The service address of the node.
```
Comment (lilin90) [inline reference/system-databases/cluster-log.md:36]:
```suggestion
* `INSTANCE`: The service address of the instance.
```

File: reference/system-databases/cluster-log.md [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+* `LEVEL`: The log level.
+* `MESSAGE`: The log content.
+
+> **Note:**
+>
+> + All fields of the cluster log table are pushed down to the corresponding node for execution. So to reduce the overhead of using the cluster log table, specify as many conditions as possible. For example, the `select * from cluter_log where instance='tikv-1'` statement only executes the log search on the `tikv-1` node.
```
Comment (lilin90) [inline reference/system-databases/cluster-log.md:42]: ditto. @reafans PTAL It seems that it should be `instance`. @TomShawn Please pay attention to all `node`s in this PR after @reafans confirms.

File: reference/system-databases/cluster-log.md [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+| 2020/03/27 15:39:37.141 | tidb | 172.16.5.40:4008 | INFO  | [ddl.go:489] ["[ddl] DDL job is finished"] [jobID=58]                                                                                                                                                                                                                                                                                       |
++-------------------------+------+------------------+-------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+```
+
+The above query results show the following process:
+
+1. The request with a DDL JOB ID of `58` is sent to the `172.16.5.40: 4008` TiDB node.
```
Comment (lilin90) [inline reference/system-databases/cluster-log.md:71]:
```suggestion
1. The request with a DDL JOB ID of `58` is sent to the `172.16.5.40: 4008` TiDB instance.
```

File: reference/system-databases/cluster-log.md (hunk 6 above)
Comment (TomShawn) [reply reference/system-databases/cluster-log.md:42]: OK

File: reference/system-databases/cluster-load.md (hunk 2 above)
Comment (reafans) [reply reference/system-databases/cluster-load.md:33]: We have tested some place in sql-diagnosis and fixed some unreasonable detail recently. And i will check and update the Chinese version recently。

File: reference/system-databases/cluster-load.md (hunk 3 above)
Comment (reafans) [reply reference/system-databases/cluster-load.md:9]: It is instance here.

File: reference/system-databases/cluster-log.md (hunk 6 above)
Comment (reafans) [reply reference/system-databases/cluster-log.md:42]: 1. The format of instance is `ip:port`, which represent a TiDB or TiKV instance.
2. The node is not accurate, maybe host is better, it represents a physical node, which contains some instance, and its format like `ip`.
3. All the node in this pr need be changed to instance.

Comment (TomShawn) [reply reference/system-databases/cluster-log.md]: All `node` is replaced in the recently added system table documents.

**Comment by TomShawn** [comment]
@reafans @lilin90 Comments addressed, PTAL again, thanks!

File: reference/system-databases/cluster-load.md [hunk 8]
```diff
@@ -0,0 +1,65 @@
...
+
+The `CLUSTER_LOAD` cluster load table provides the current load information of the server where each instance of the TiDB cluster is located.
+
+{{< copyable "sql" >}}
+
+```sql
+desc cluster_load;
+```
+
+```
```
Comment (lilin90) [inline reference/system-databases/cluster-load.md:17]: Adding `sql` to the results can also make it looks better with colors at the website.
```suggestion
```sql
```

File: reference/system-databases/cluster-load.md [hunk 9]
```diff
@@ -0,0 +1,65 @@
...
+
+The following example shows how to query the current load information of cpu using the `CLUSTER_LOAD` table:
+
+{{< copyable "sql" >}}
+
+```sql
+select * from cluster_load where device_type='cpu' and device_name='cpu';
+```
+
+```
```
Comment (lilin90) [inline reference/system-databases/cluster-load.md:51]:
```suggestion
```sql
```

File: reference/system-databases/cluster-log.md [hunk 10]
```diff
@@ -0,0 +1,73 @@
...
+title: CLUSTER_LOG
+summary: Learn the `CLUSTER_LOG` cluster log table.
+category: reference
+---
+
+# CLUSTER_LOG
+
+You can query cluster logs on the `CLUSTER_LOG` cluster log table. By pushing down query conditions to each instance, the impact of the query on cluster performance is less than that of the `grep` command.
+
+To get the logs of the TiDB cluster before v4.0, you need to log in to each instance to summarize logs. This cluster log table in 4.0 provides the global and time-ordered log search result, which makes it easier to track full-link events. For example, by searching logs according to the `region id`, you can query all logs in the life cycle of this Region; similarly, by searching the full link log through the slow log's `txn id`, you can query the flow and the number of keys scanned by this transaction at each instance.
```
Comment (lilin90) [inline reference/system-databases/cluster-log.md:11]:
```suggestion
To get the logs of the TiDB cluster before v4.0, you need to log in to each instance to summarize logs. This cluster log table in 4.0 provides the global and time-ordered log search result, which makes it easier to track full-link events. For example, by searching logs according to the `region id`, you can query all logs in the life cycle of this Region. Similarly, by searching the full link log through the slow log's `txn id`, you can query the flow and the number of keys scanned by this transaction at each instance.
```

File: reference/system-databases/cluster-log.md [hunk 11]
```diff
@@ -0,0 +1,73 @@
...
+
+{{< copyable "sql" >}}
+
+```sql
+desc cluster_log;
+```
+
+```
```
Comment (lilin90) [inline reference/system-databases/cluster-log.md:19]:
```suggestion
```sql
```

File: reference/system-databases/cluster-log.md [hunk 12]
```diff
@@ -0,0 +1,73 @@
...
+
+The following example shows how to query the execution process of a DDL statement using the `CLUSTER_LOG` table:
+
+{{< copyable "sql" >}}
+
+```sql
+select * from `CLUSTER_LOG` where message like '%ddl%' and message like '%job%58%' and type='tidb' and time > '2020-03-27 15:39:00';
+```
+
+```
```
Comment (lilin90) [inline reference/system-databases/cluster-log.md:54]:
```suggestion
```sql
```

File: reference/system-databases/cluster-systeminfo.md [hunk 13]
```diff
@@ -0,0 +1,54 @@
...
+
+You can use the `CLUSTER_SYSTEMINFO` kernel parameter table to query the kernel configuration information of the server where all nodes of the cluster are located. Currently, you can query the information of the `sysctl` system.
+
+{{< copyable "sql" >}}
+
+```sql
+desc cluster_systeminfo;
+```
+
+```
```
Comment (lilin90) [inline reference/system-databases/cluster-systeminfo.md:17]:
```suggestion
```sql
```

File: reference/system-databases/cluster-systeminfo.md [hunk 14]
```diff
@@ -0,0 +1,54 @@
...
+* `NAME`: The configuration name corresponding to `sysctl`.
+* `VALUE`: The value of the configuration item corresponding to `sysctl`.
+
+The following example shows how to query the kernel version of all servers in the cluster using the `CLUSTER_SYSTEMINFO` system information table.
+
+```sql
+select * from CLUSTER_SYSTEMINFO where name like '%kernel.osrelease%'
+```
+
+```
```
Comment (lilin90) [inline reference/system-databases/cluster-systeminfo.md:46]:
```suggestion
```sql
```

**Pull request merged**